slang.h

```diff
@@ -668,6 +668,8 @@ extern "C"
 #define SLANG_E_NOT_FOUND                   SLANG_MAKE_CORE_ERROR(5)
     //! An unhandled internal failure (typically from unhandled exception)
 #define SLANG_E_INTERNAL_FAIL               SLANG_MAKE_CORE_ERROR(6)
+    //! Could not complete because some underlying feature (hardware or software) was not available 
+#define SLANG_E_NOT_AVAILABLE               SLANG_MAKE_CORE_ERROR(7)
```

Comment:
Does this PR depend on the code from the previous PR? Otherwise having these extra diffs here is a bit confusing.

Reply:
It is dependent on previous PR... maybe should have rebased or done a merge after the previous PR was merged. I did it cos I wanted to test the code I added to make tests run only if features are available worked on CI.